components/brave_wallet/browser/cardano/cardano_max_send_solver.cc

```diff
@@ -30,27 +30,21 @@ base::expected<CardanoTransaction, std::string> CardanoMaxSendSolver::Solve() {
   DCHECK(base_transaction_.TargetOutput());
   DCHECK(base_transaction_.sending_max_amount());
 
-  auto result = base_transaction_;
-  result.AddInputs(std::move(inputs_));
+  auto tx_with_inputs = std::move(base_transaction_);
```

Comment:
This brings this class object to invalid state. There are DCHECKS, but Solve method still can be double-time executed on release. We should add explicit protection here if we don't want this method to be double-time executed,

Reply:
It's never going to be reused. Removed `std::move`